src/test/java/com/hocs/test/glue/CaseDecisionStepDefs.java

```diff
@@ -94,7 +103,7 @@ public void iCanOnlySelectFromAFixedListOfAnswering(String arg) {
 
     @When("^I close the case with a decision of \"([^\"]*)\"$")
     public void iCloseTheCaseWithADecisionOf(String status) {
-        page.getCaseId();
+        markUpDecision.getCaseId();
```

Comment:
if markUpDecision is a page, rename the class so that the suffix is page, this will make it easier to understand what page is doing what.

Top of file declaration:
```
MarkUpDecisionPage = markUpDecisionPage;
```

useage:
```
markUpDecisionPage.getCaseId();
```

Reply:
do this for all the other pages that you have which do not end with Page, or create a backlog task to do it and pick it up later